core/src/main/java/com/sequenceiq/cloudbreak/service/altus/AltusMachineUserService.java

```diff
@@ -104,7 +104,7 @@ public void cleanupMachineUser(String machineUserName, String accountId) {
     public DataBusCredential getOrCreateDataBusCredentialIfNeeded(Long stackId) throws IOException {
         LOGGER.debug("Get or create databus credential for stack");
         Stack stack = stackService.get(stackId);
-        Cluster cluster = clusterService.getCluster(stack);
+        Cluster cluster = clusterService.findOneByStackIdOrNotFoundError(stack.getId());
```

Comment:
Nit: `stack.getId()` ideally matches `stackId`, so it could perhaps be simplified.